sites/skeleton.dev/src/lib/__tests__/Tabs.test.js

```diff
@@ -0,0 +1,45 @@
+import { render, fireEvent } from '@testing-library/svelte';
```

Comment:
Once again, please reference our existing components. We never separate our test cases like this. They are paired directly with the component in the component packages. Never on the doc app.

We cover this is the contribution docs:
https://www.skeleton.dev/docs/resources/contribute/components#add-components

And `Tabs.test.ts` already exists in the component directory:
https://github.com/skeletonlabs/skeleton/tree/main/packages/skeleton-svelte/src/components/Tabs